plugins/package-managers/bazel/src/main/kotlin/BazelModel.kt

```diff
@@ -31,10 +31,10 @@ private val json = Json { ignoreUnknownKeys = true }
  */
 @Serializable
 internal data class Lockfile(
-    val flags: Flags
+    val flags: Flags? = null
```

Comment:
From the perspective of Bazel, `flags` in the lock file is "duplicated information", but for the `Bazel` package manager plugin, it is currently the only way to determine where to retrieve package metadata.

So the benefit of this change is that the analyzer will continue to work for Bazel projects that use a version >= 7.2 and the Bazel Central Registry (since it is the default in `RemoteBazelRegistryService`). However, the current implementation will not work for projects that use Bazel >= 7.2 with a local registry for example. One way to address that might be to look for the `--registry` flag in a `.bazelrc` file.

This is still a very useful improvement, if not a fix, with no downsides so should probably be merged as is. In that case we might want to create an issue to track the above mentioned limitation.

Reply:
Yes, the first commit only makes ORT "not crash" on >= 7.2 lock files, regardless of local or remote registries.  Ńothing more.

I am in discussion with Bazel people on slack and collecting the alternatives to using the `flags` property. As soon as I have enough info, I will create an issue for it.